src/super_context.ml

```diff
@@ -135,21 +135,37 @@ let expand_ocaml_config t pform name =
       "Unknown ocaml configuration variable %S"
       name
 
-let expand_var t ~scope ~bindings pform syntax_version =
+let expand_env ~env pform s : Value.t list =
+  match String.rsplit2 s ~on:'=' with
+  | None ->
+    Errors.fail (String_with_vars.Var.loc pform)
+      "%s must always come with a default value"
```

Comment:
I would add a hint so that the user doesn't have to look in the manual for the syntax:

```
Hint: the syntax is: %{env:VAR=DEFAULT-VALUE}
```